### PR TITLE
Update the following RL notebooks to Ray 0.8.5:

### PR DESCRIPTION
**Pull request by goelakash**
*Issue #, if available:*

*Description of changes:*
rl_bin_packing_ray_custom.ipynb
rl_news_vendor_ray_custom.ipynb

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by yijiezh** [comment]
LGTM if the notebooks are tested

File: reinforcement_learning/rl_resource_allocation_ray_customEnv/rl_bin_packing_ray_custom.ipynb [hunk 1]
```diff
@@ -191,6 +191,30 @@
     "    !/bin/bash ./common/setup.sh"
```
Comment (yijiezh) [inline reinforcement_learning/rl_resource_allocation_ray_customEnv/rl_bin_packing_ray_custom.ipynb:191]: <p>This is not a custom image. We should just let the RLEstimator pointing to this image</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1799/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

File: reinforcement_learning/rl_resource_allocation_ray_customEnv/rl_news_vendor_ray_custom.ipynb [hunk 2]
```diff
@@ -184,6 +184,30 @@
     "    !/bin/bash ./common/setup.sh"
```
Comment (yijiezh) [inline reinforcement_learning/rl_resource_allocation_ray_customEnv/rl_news_vendor_ray_custom.ipynb:184]: <p>same as above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1799/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

File: reinforcement_learning/rl_resource_allocation_ray_customEnv/rl_bin_packing_ray_custom.ipynb (hunk 1 above)
Comment (goelakash) [reply reinforcement_learning/rl_resource_allocation_ray_customEnv/rl_bin_packing_ray_custom.ipynb]: Right, fixed.

File: reinforcement_learning/rl_resource_allocation_ray_customEnv/rl_news_vendor_ray_custom.ipynb (hunk 2 above)
Comment (goelakash) [reply reinforcement_learning/rl_resource_allocation_ray_customEnv/rl_news_vendor_ray_custom.ipynb]: fixed

**Comment by goelakash** [comment]
> LGTM if the notebooks are tested

The tests are passing, so I think this should be good to merge.

**Review by yijiezh — COMMENTED** [review]
Can you confirm that all the notebooks are working with the updated version. I've tested the automatic tuning one, and that does not work

File: reinforcement_learning/common/misc.py [hunk 3]
```diff
@@ -141,3 +141,6 @@ def get_execution_role(role_name="sagemaker", aws_account=None, aws_region=None)
     return client.get_role(RoleName=role_name)['Role']['Arn']
 
 
+def wait_for_training_job_to_complete(job_name):
```
Comment (yijiezh) [inline reinforcement_learning/common/misc.py:144]: I am not quite sure if we need this function. I guess plotting should be able to done without waiting the training job to be finish?

Comment (goelakash) [reply reinforcement_learning/common/misc.py:144]: No, the function errors out for plotting if the job is not complete. Tested in all notebooks and the behaviour was pretty consistent.

File: reinforcement_learning/rl_unity_ray/Dockerfile [hunk 4]
```diff
@@ -11,8 +11,6 @@ RUN pip install --upgrade \
     gym-unity \
     mlagents-envs
 
-RUN pip install sagemaker-containers --upgrade
```
Comment (yijiezh) [inline reinforcement_learning/rl_unity_ray/Dockerfile:14]: any reason to remove to this line?

Comment (goelakash) [reply reinforcement_learning/rl_unity_ray/Dockerfile]: Thanks, this was part of the previous change. Removed.

**Review by yijiezh — APPROVED** [review]
LGTM

**Pull request merged**